packages/react-server/core/__tests__/integration/redirectForward/RedirectForwardSpec.js

```diff
@@ -1,12 +1,12 @@
-var helper = require("../specRuntime/testHelper"),
+var helper = require("../../../test/specRuntime/testHelper"),
```

Comment:
> What if we moved the test helpers into a separate package? Then they could be required without worrying about relative location.
I had that thought too, but it seems like a weird thing to publish a separate module for? I'm not opposed to it though.